lib/command/init.js

```diff
@@ -245,6 +254,33 @@ module.exports = function (initPath) {
         }
       }
 
+      if (installGitHubActions === true) {
+        const ghaYml = `name: Run Tests
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js 14.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14
+      - run: npx playwright install-deps
+      - run: npm i
+      - run: npm run test
```

Comment:
I would prefer to use `npx codeceptjs run` here to ensure that CodeceptJS is executed